src/i18n.js

```diff
@@ -208,6 +208,31 @@ const resources = {
       select: 'Вибір мови',
     },
   },
+  'de-DE': {
+    translation: {
+      title: 'WiFi Card',
+      'desc.use':
+        'Druck dir eine simple Karte mit deinen WiFi Anmeldedaten. Klebe sie an deinen Kühlschrank, behalte sie in deinem Portemonnaie, etc.',
+      'desc.privacy':
+        'Deine WiFi Anmeldedaten werden niemals zum Server gesendet. Es gibt kein Tracking, Analytics, oder Fingerprinting auf dieser Website. Hier geht es zum',
+      'desc.source': 'Source Code',
```

Comment:
```suggestion
      'desc.source': 'Quellcode',
```

Reply:
Ahh yes the good old fashioned denglish problem i will add this and some of the other suggestions ✌🏼